docs/scos/dev/feature-integration-guides/202307.0/glue-api/data-exchange-api/install-the-data-exchange-api.md

```diff
@@ -147,19 +171,21 @@ console transfer:generate
 
 {% info_block warningBox "Verification" %}
 
-Ensure that you've triggered the following changes by checking the database:
+Make sure you've triggered the following changes by checking the database:
 
 | DATABASE ENTITY | TYPE |
 | --- | --- |
 | spy_dynamic_entity_configuration | table |
 
-Add configurations for dynamic entities. In order to do that follow the instructions here [how to configure Dynamic Data API](/docs/scos/dev/glue-api-guides/{{page.version}}/dynamic-data-api/how-to-guides/how-to-configure-dynamic-data-api.html)  
+Add configurations for dynamic entities. In order to do that follow the instructions here [How to configure Data Exchange API](/docs/scos/dev/glue-api-guides/{{page.version}}/data-exchange-api/how-to-guides/how-to-configure-data-exchange-api.html)  
```

Comment:
Why is this part of verification?

Reply:
You're right. I guess this part can be extracted from the block